integration_tests/e2e/app.cy.ts

```diff
@@ -1,6 +1,6 @@
 import IndexPage from '../pages'
 import Page from '../pages/page'
-import helpers from '../support/helpers'
+import Helpers from '../support/helpers'
```

Comment:
I've found this commit (015c53af) a bit hard to review as it's changing so many files into classes in one go.

With the amount that's changing here, it's been a bit tricky to review carefully, but the approach looks good me and luckily we have a comprehensive test suite. With future changes like this that affect a big chunk of the codebase more structurally than reordering some keys etc, moving one class at a time would probably lighten the load as a reviewer.

No need to change anything this time, as rebasing will be a pain, but just something to bear in mind for next time!